### PR TITLE
Added ability to deploy home fss separately from /data fss.

### PR DESCRIPTION
**Pull request by andriy-safe-ai**

### CODE REVIEW

File: fss.tf
Comment (xksteven) [inline fss.tf:1]: Why did you delete these?

File: fss_home.tf
Comment (xksteven) [inline fss_home.tf:1]: I don't understand why make a new file for this?

File: playbooks/site.yml [hunk 1]
```diff
@@ -83,20 +83,20 @@
   become: true
   vars:
     tmp_home: "/tmp/home_tmp/"
-    remote_path: "{% if create_fss|bool %}/home{% else%}{{nfs_source_path}}{% endif %}"
+    remote_path: "{% if create_home_fss|bool %}/home{% else%}{{home_fss_source_path}}{% endif %}"
   tasks:
     - include_role:
         name: nfs-client
```
Comment (xksteven) [inline playbooks/site.yml:89]: Is NFS false? I'm asking because your removing the condition that checks add_nfs

**Review by xksteven — COMMENTED** [review]
Where does it prevent the /data from getting created now? I thought you mentioned that

File: fss.tf
Comment (andriy-safe-ai) [reply fss.tf:1]: I moved the Terraform resources for the home FSS to a separate file. I thought that it might be neater but now I'm having second thoughts. I'll combine `fss.tf` and `fss_home.tf`.

File: fss_home.tf
Comment (andriy-safe-ai) [reply fss_home.tf:1]: Thought it would be cleaner. I've changed my mind. It'll combine `fss.tf` and `fss_home.tf`.

File: playbooks/site.yml (hunk 1 above)
Comment (andriy-safe-ai) [reply playbooks/site.yml:89]: `add_nfs` needs to be true to deploy `/data/` FSS making it a dependency for `/home` FSS. Since we want to decouple `/data/` and `/home` we need to be able to deploy `/home` FSS without `/data/` we no loner want to check whether `add_nfs` is true.

**Comment by andriy-safe-ai** [comment]
> Where does it prevent the /data from getting created now? I thought you mentioned that

We aren’t preventing /data from being deployed. I’m adding functionality that allows us to deploy an FSS for /home without requiring an FSS for /data. In fss.tf and fss_home.tf, you’ll see that deploying an FSS for /home now only checks if the create_home_fss variable is true, and no longer depends on create_fss being true.

We can still create the FSS for /data by setting create_fss to true. Additionally, we have new variables (create_home_fss, home_fss_source_IP, and home_fss_source_path) to provision and configure an FSS for /home separately. This means we can set create_fss to false and create_home_fss to true, deploying an FSS for /home without deploying one for /data.

Let me know if that explains it better.

**Review by steven-basart — APPROVED** [review]
LGTM

**Comment by andriy-safe-ai** [comment]
Rebased to one commit and merged.
